internal/controller/apiextensions/composite/reconciler.go

```diff
@@ -421,7 +442,7 @@ func NewReconciler(mgr manager.Manager, of resource.CompositeKind, opts ...Recon
 		log:    logging.NewNopLogger(),
 		record: event.NewNopRecorder(),
 
-		pollInterval: defaultPollInterval,
+		pollInterval: func(_ context.Context, _ *composite.Unstructured) time.Duration { return defaultPollInterval },
```

Comment:
Note this default won't actually be used. In practice we plumb down `--poll-interval` from the CLI using `WithPollInterval`.